### PR TITLE
FOIA-244: Inline documentation and README for foia_workflow module.

### PR DESCRIPTION
**Pull request by jesseday**

### CODE REVIEW

File: docroot/modules/custom/foia_workflow/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+
+INSTALLATION
+------------
+
+FOIA Workflow is a custom Drupal module so unlike contrib modules, the codebase
+is not installed via composer. Enable as you would normally enable a
+contributed Drupal module. Visit
+https://www.drupal.org/docs/8/extending-drupal-8/installing-drupal-8-modules#s-step-2-enable-the-module
+for futher information.
```
Comment (bemarlan) [inline docroot/modules/custom/foia_workflow/README.md]: Spelling correction to "further"

File: docroot/modules/custom/foia_workflow/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+INTRODUCTION
+------------
+
+The FOIA Workflow module adds custom access restrictions and grants to Annual
+FOIA Report Data nodes based on a user's agency and role, and the workflow
+state of the node.
+
+
+### Additional access restrictions
```
Comment (bemarlan) [inline docroot/modules/custom/foia_workflow/README.md]: I'm thinking we should remove the word "Additional" from both of these headings.

File: docroot/modules/custom/foia_workflow/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+The FOIA Workflow module adds custom access restrictions and grants to Annual
+FOIA Report Data nodes based on a user's agency and role, and the workflow
+state of the node.
+
+
+### Access restrictions
+
+ * Users with the role Agency Administrator are not allowed to edit a report
+  that is authored by another user if it is still in a `draft` state.
```
Comment (bemarlan) [inline docroot/modules/custom/foia_workflow/README.md]: > if it is still in a `draft` state.

Should be

if it is in a `draft` or `back_with_agency` state.

Comment (jesseday) [reply docroot/modules/custom/foia_workflow/README.md]: @bemarlan I think the readme is true to what's happening in `foia_workflow_node_access`.  Does that access check need to be updated to include the `back_with_agency` state?

Comment (bemarlan) [reply docroot/modules/custom/foia_workflow/README.md]: No, this can be disregarded in this module.

**Pull request merged**